src/interpreter/ByteCode.h

```diff
@@ -57,12 +57,11 @@ class ByteCode {
     {
     }
 
-    virtual size_t byteCodeSize()
-    {
-        return sizeof(ByteCode);
-    }
 #endif
 
+    size_t getBaseSize();
+    size_t getSize(FunctionType* function);
```

Comment:
Add `ASSERT(getBaseSize() >= sizeof(ByteCode));` to the constructor. It checks that 0 is not used for valid opcodes.

Reply:
It is implemented.